tests/e2e/common.go

```diff
@@ -397,3 +402,163 @@ func getUpgradeStrategy() config.USDEUserStrategy {
 		return userStrategy
 	}
 }
+
+// test suite common functions
+var _ = BeforeSuite(func() {
+
+	var err error
+	// make output directory to store temporary outputs; if it's there from before delete it
+	disableTestArtifacts = os.Getenv("DISABLE_TEST_ARTIFACTS")
+	// pod logs env
+	enablePodLogs = os.Getenv("ENABLE_POD_LOGS")
+	if disableTestArtifacts != "true" {
+		setupOutputDir()
+	}
+
+	openFiles = make(map[string]*os.File)
+
+	stopCh = make(chan struct{})
+
+	upgradeStrategy = getUpgradeStrategy()
+	Expect(upgradeStrategy.IsValid()).To(BeTrue())
```

Comment:
I guess we should consider if every test requires this environment variable to be specified. Some tests will only be for one strategy, so it will be nice if user doesn't necessarily need to specify it.

Reply:
Perhaps we can just have a reusable function which each suite's `BeforeSuite()` calls which contains everything but that?

Reply:
After we have created the appropriate test suites for each strategy, I don't believe it will be necessary but for now it is.